src/app/space/create/deployments/apps/deployment-details.component.ts

```diff
@@ -241,4 +244,39 @@ export class DeploymentDetailsComponent {
     );
     return latch;
   }
+
+  private getTooltipContents(): any {
+    return {
+      contents: (d: any) => {
+        // d is an object containing the data displayed for a given data point in the tooltip
+        // example: [{ x: Date, value: number, id: string, index: number, name: string }]
+        // http://c3js.org/reference.html#tooltip-contents
+        let tipRows: string = '';
+        let color = '#0088ce'; // pf-blue-400
+        let units: string = '';
+        if (d[0].name === 'CPU') {
+          units = this.cpuConfig.units;
+        } else if (d[0].name === 'Memory') {
+          units = this.memUnits;
+        }
+        tipRows += '<tr><th colspan="2">' + d[0].x.toLocaleString() + '</th></tr>';
+        tipRows +=
+        '<tr>' +
+        '  <td class="name"><span style="background-color:' + color + '"></span>' + d[0].name + '</td>' +
+        '  <td class="value text-nowrap">' + d[0].value + ' ' + units + '</td>' +
+        '</tr>';
+        return this.getTooltipTableHTML(tipRows);
+      }
+    };
+  }
+
+  private getTooltipTableHTML(tipRows: string): string {
+    return '<div class="module-triangle-bottom">' +
```

Comment:
why not use the back tick for this, it would be much cleaner and easier to maintain

Reply:
Sure that sounds good, I've updated this to use back ticks instead.